### ISSUE TITLE
Poor performance when using rows to scan in data

### ISSUE THREAD

**Issue by jimmiebtlr** [go-jet/jet#119]
**Describe the bug**
When using rows to read row by row, the performance is around 10x worse than using raw sql. 

Note, I also gave it a try using raw sql, and using qrm.ScanOneRowToDest, with the same poor performance seen there.  

Some additional background
- Using a struct with 3 not huge jet models (total probably 30 fields, 10 of which are duplicate with an AS clause, roughtly 10 fields used total out of them).
- Reducing fields to 1 model with 11 fields cuts the time 50%
- I'm returning 1M rows.
- Similar code that uses Query or QueryContext instead of Rows is returning in around 25% the time.

My guess is the overhead of reflecting on every row is driving this behavior.

**Environment (please complete the following information):**
 - OS: macosx
 - Database: mysql
-  Database driver: mysql
 - Jet version 2.5

**Code snippet**
```
q, args := query.Sql()
rows, err := db.Query(q, args...)
// handle err and defer close

for rows.Next() {
	rowData := generator()
	err = qrm.ScanOneRowToDest(rows, rowData)
	// handle err
}
```

**Expected behavior**
I expect the results to take < 40% current time.

**Comment by go-jet** [comment]
Hi @jimmiebtlr. Managed to reproduce it. New scanContext is unnecessarily created with each `Scan` call. I've made a fix on the `rows-perf` branch. Rows scan should now be several times faster.

**Comment by jimmiebtlr** [comment]
Tested rows-perf branch and performance is much improved thanks!!

**Comment by go-jet** [comment]
Fixed with [Release 2.7.1](https://github.com/go-jet/jet/releases/tag/v2.7.1)
